packages/api/package.json

```diff
@@ -0,0 +1,47 @@
+{
+    "name": "@comet-brevo-module/api",
+    "version": "1.0.0",
+    "repository": {
+        "type": "git",
+        "url": "https://github.com/vivid-planet/comet-brevo-module/",
+        "directory": "packages/api/"
+    },
+    "license": "BSD-2-Clause",
+    "main": "lib/index.js",
+    "types": "lib/index.d.ts",
+    "bin": {
+        "comet-api-generator": "bin/comet-api-generator.js"
+    },
+    "files": [
+        "bin/**/*.js",
+        "lib/*"
+    ],
+    "scripts": {
+        "clean": "rimraf lib",
+        "build": "$npm_execpath run clean && tsc -p tsconfig.build.json",
+        "dev": "tsc --watch --preserveWatchOutput -p tsconfig.build.json",
+        "lint": "run-p lint:eslint lint:tsc",
+        "lint:eslint": "eslint --max-warnings 0 src/ package.json",
+        "lint:tsc": "tsc --noEmit",
+        "list-exports": "ts-node list-exports.ts",
+        "test": "jest",
+        "test:watch": "jest --watch"
+    },
+    "dependencies": {
+        "rimraf": "^3.0.0"
+    },
+    "devDependencies": {
+        "@comet/eslint-config": "^5.3.0",
+        "@nestjs/common": "^9.0.0",
```

Comment:
You'll probably need some peerDependencies here

Reply:
As of now this works without further peerDependencies

Reply:
`@nestjs/common`, `reflect-metadata` and `rxjs` should be both peer dependencies (for projects) and dev dependencies (for development).

Reply:
I added those a peers in  [9a54cbc](https://github.com/vivid-planet/comet-brevo-module/pull/3/commits/9a54cbcfed81492e19850a0cbf2779887f3414f5)